### PR TITLE
[Lagos-2022] Initial event stub

### PR DESCRIPTION
**Pull request by haroldsphinx**

### CODE REVIEW

File: data/events/2022-lagos.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
+name: "2022-lagos" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2022" # The year of the event. Make sure it is in quotes.
+city: "Lagos" # The displayed city name of the event. Capitalize it.
+event_twitter: "devopsdayslag" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
+description: "Devopsdays is coming to Lagos!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+cancel: "true"
```
Comment (phrawzty) [inline data/events/2022-lagos.yml:7]: Your event won't appear on the website it this is set to `true`.

```suggestion
cancel: "false"
```

File: data/events/2022-lagos.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+# All dates are in unquoted 2020-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2020-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate:  # The start date of your event. Leave blank if you don't have a venue reserved yet.
```
Comment (phrawzty) [inline data/events/2022-lagos.yml:14]: It's OK if you don't have the exact date yet. I understand that you're thinking of either 29 October or 5 November. What needs to be resolved on your end in order to lock a specific date into place?

File: data/events/2022-lagos.yml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  - name: sponsor
+  - name: contact
+  - name: conduct
+# - name: example
+#   icon: "map-o" # This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
```
Comment (phrawzty) [inline data/events/2022-lagos.yml:52]: Note to admins: the DL is up to date.

**Review by phrawzty — CHANGES_REQUESTED** [review]

File: data/events/2022-lagos.yml (hunk 2 above)
Comment (haroldsphinx) [reply data/events/2022-lagos.yml:14]: We have finally been able to conclude on this to be October 29th, I have updated it in the code as well

File: data/events/2022-lagos.yml [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+# All dates are in unquoted 2020-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2020-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: October 29th, 2022 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: October 29th, 2022 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: August 1st, 2022 # start accepting talk proposals.
```
Comment (somatorio) [inline data/events/2022-lagos.yml:18]: Both should be in the following format: YYYY-MM-DDTHH:MM:SS+TZ:TZ

Example: 2022-10-29T08:00:00+01:00

File: data/events/2022-lagos.yml [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+description: "Devopsdays is coming to Lagos!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+cancel: "false"
+
+# All dates are in unquoted 2020-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2020-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: October 29th, 2022 # The start date of your event. Leave blank if you don't have a venue reserved yet.
```
Comment (somatorio) [inline data/events/2022-lagos.yml:14]: All dates should be in the following format: YYYY-MM-DDTHH:MM:SS+TZ:TZ

Example: 2022-10-29T08:00:00+01:00

File: data/events/2022-lagos.yml [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+registration_date_start: August 1st, 2022 # start accepting registration. Leave blank if registration is not open yet
+registration_date_end: October 28th, 2022 # close registration. Leave blank if registration is not open yet.
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+coordinates: "6.54451° N, 3.39226° E" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
```
Comment (somatorio) [inline data/events/2022-lagos.yml:32]: Should be "6.5410064, 3.3970042"

File: data/events/2022-lagos.yml [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+    image: "ewere.jpg"
+  - name: "Idowu Emehinola"
+    employer: "Advanced Blockchain"
+    twitter: "@_hydeenoble"
+    github: "hydeenoble"
+    image: "idowu.jpg"
+  - name: AbdulBasit Kabir
+    employer: "Interswitch"
+
+organizer_email: "ga" # Put your organizer email address here
```
Comment (somatorio) [inline data/events/2022-lagos.yml:72]: Should be lagos@devopsdays.org

File: static/_redirects [hunk 8]
```diff
@@ -1,123 +1,118 @@
-/amsterdam/*		/events/2022-amsterdam/:splat		302
+/amsterdam/*		/events/2020-amsterdam/:splat		302
```
Comment (somatorio) [inline static/_redirects:1]: Your PR should not change other devopsdays redirects... I suspect that your fork is outdated, maybe you need to run `git pull upstream main --rebase`? ([Provided you set the upstream origin](https://github.com/devopsdays/devopsdays-web/blob/main/CONTRIBUTING.md#setting-your-fork-up-the-first-time))

File: static/_redirects 2 [hunk 9]
```diff
@@ -0,0 +1,165 @@
+/amsterdam/*		/events/2022-amsterdam/:splat		302
```
Comment (somatorio) [inline static/_redirects 2:1]: I don't think this files should be here...?

**Review by somatorio — CHANGES_REQUESTED** [review]
There's a lot of files renaming and some *.icloud files that should be looked into, all into other cities, was that some script or something like this?

**Pull request closed without merging**